### PR TITLE
Replace retail pipeline with new notebook file

### PR DESCRIPTION
**Pull request by YanxuanLiu**

### CODE REVIEW

**Comment by nvliyuan** [comment]
can you help clear the error info in retail-datagen.ipynb? thx
```
---------------------------------------------------------------------------
ModuleNotFoundError                       Traceback (most recent call last)
Cell In[1], line 2
      1 import os
----> 2 import pandas as pd
      3 import multiprocessing as mp
      4 import random

ModuleNotFoundError: No module named 'pandas'
```

**Comment by YanxuanLiu** [comment]
> can you help clear the error info in retail-datagen.ipynb? thx
> 
> ```
> ---------------------------------------------------------------------------
> ModuleNotFoundError                       Traceback (most recent call last)
> Cell In[1], line 2
>       1 import os
> ----> 2 import pandas as pd
>       3 import multiprocessing as mp
>       4 import random
> 
> ModuleNotFoundError: No module named 'pandas'
> ```

Sure, cleared all output

File: examples/SQL+DF-Examples/retail-analytics/notebooks/python/retail-datagen.ipynb [hunk 1]
```diff
@@ -19,85 +21,92 @@
     "import random\n",
     "\n",
     "# You need to update these to your real paths!\n",
-    "dataRoot = os.getenv(\"DATA_ROOT\", 'gs://bucket-name/data/raw/')"
+    "dataRoot = os.getenv(\"DATA_ROOT\", '/path/to/your/datasets')"
```
Comment (NvTimLiu) [inline examples/SQL+DF-Examples/retail-analytics/notebooks/python/retail-datagen.ipynb:24]: Shall we upload the sample dataset somewhere like https://github.com/NVIDIA/spark-rapids-examples/tree/main/datasets?

File: examples/SQL+DF-Examples/retail-analytics/notebooks/python/retail-analytic.ipynb [hunk 2]
```diff
@@ -93,22 +91,22 @@
     "        return spark.read.format(format).load(file_path)\n",
     "\n",
     "# read sales data\n",
```
Comment (NvTimLiu) [inline examples/SQL+DF-Examples/retail-analytics/notebooks/python/retail-analytic.ipynb:93]: Are there any ordering requirements for retail analytics and data generation?

File: examples/SQL+DF-Examples/retail-analytics/notebooks/python/retail-datagen.ipynb (hunk 1 above)
Comment (YanxuanLiu) [reply examples/SQL+DF-Examples/retail-analytics/notebooks/python/retail-datagen.ipynb:24]: No, in `retail-datagen` we only need to declare the `dataRoot`, the notebook will generate data under `dataRoot`. These data will be used in `retail-analytic`

File: examples/SQL+DF-Examples/retail-analytics/notebooks/python/retail-analytic.ipynb (hunk 2 above)
Comment (YanxuanLiu) [reply examples/SQL+DF-Examples/retail-analytics/notebooks/python/retail-analytic.ipynb:93]: Yes, data generation -> analytics. I will fix the order in pipeline param.

File: examples/SQL+DF-Examples/retail-analytics/notebooks/python/retail-analytic.ipynb [hunk 3]
```diff
@@ -3,50 +3,46 @@
   {
    "cell_type": "code",
    "execution_count": null,
+   "id": "12f4c0ee",
```
[the remark this answers is not in the document]
Comment (YanxuanLiu) [reply examples/SQL+DF-Examples/retail-analytics/notebooks/python/retail-analytic.ipynb]: Not necessary, Jupyter will assign id to cells automatically when executing. Removed these ids from notebook.

**Review by NvTimLiu — APPROVED** [review]
LGTM, +1

**Pull request merged**